lib/glue/mounters/azdo_mounter.rb

```diff
@@ -0,0 +1,54 @@
+require 'glue/mounters/base_mounter'
+require 'fileutils'
+
+class Glue::AzDOMounter < Glue::BaseMounter
+
+  Glue::Mounters.add self
+
+  def initialize trigger, options
+    super(trigger)
+    @options = options
+    @name = "AzDO Git"
+    @description = "Pull a repo."
+  end
+
+  def mount target
+    base = @options[:working_dir]
+
+    Glue.debug "Making base."
+    FileUtils.mkdir_p base
+
+    # Grab the path used as git url, excluding PAT.
+    # format of this target must be https://DUMMY:<PAT_TOKEN>@dev.azure.com/<account>/<project>/_git/<repository>
+    protocol, azdo_domain, account, project, repository = target.match(/\A(.*\/\/).*@(.*)\/(.*)\/(.*)\/_git\/(.+?)[\/]{0,1}\z/i).captures
```

Comment:
Maybe get these parameters separately and build the URL?

Reply:
I am thinking on maybe having PATTOKEN as a configuration parameter, as can be useful for something else (working on creating work items on Azure DevOps, like the ones for Jira, Pivotal...)

Parameters I extract from the match are for building the folder name used for temporary storage, but the target url should be ````https://dev.azure.com/../../_git/....```

So that way, you can just use the Git Url you can copy/paste from Azure DevOps

Reply:
Ohh I see. Ok, so just move the PAT to paramter - which could also help in reducting it.